acceptance/api_v1_orgs_test.go

```diff
@@ -132,5 +133,42 @@ var _ = Describe("Orgs API Application Endpoints", func() {
 				Expect(string(bodyBytes)).To(Equal(""))
 			})
 		})
+
+		Describe("DELETE api/v1/orgs/:org", func() {
+			It("deletes an organization", func() {
+				response, err := Curl("DELETE", fmt.Sprintf("%s/api/v1/orgs/%s", serverURL, org),
+					strings.NewReader(``))
+				Expect(err).ToNot(HaveOccurred())
+				Expect(response).ToNot(BeNil())
+				defer response.Body.Close()
+				bodyBytes, err := ioutil.ReadAll(response.Body)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(response.StatusCode).To(Equal(http.StatusOK), string(bodyBytes))
+				Expect(string(bodyBytes)).To(Equal(""))
+
+				_, err = helpers.Kubectl(fmt.Sprintf("get namespace %s", org))
+				Expect(err).To(HaveOccurred())
+			})
+
+			It("deletes an organization including apps and services", func() {
+				app1 := newAppName()
+				makeApp(app1)
+				svc1 := newServiceName()
+				makeCustomService(svc1)
+				bindAppService(app1, svc1, org)
+
+				response, err := Curl("DELETE", fmt.Sprintf("%s/api/v1/orgs/%s", serverURL, org),
+					strings.NewReader(``))
+				Expect(err).ToNot(HaveOccurred())
+				Expect(response).ToNot(BeNil())
+				defer response.Body.Close()
+				bodyBytes, err := ioutil.ReadAll(response.Body)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(string(bodyBytes)).To(Equal(""))
+
+				_, err = helpers.Kubectl(fmt.Sprintf("get namespace %s", org))
```

Comment:
we never checked that the service lived in this namespace thus checking that a namespace doesn't exist doesn't tell us anything about the service. Imaging if someone changed the code an introduced a bug where the service appears in another namespace. This test would continue to be green, though based on the description one would expect it to fail.

Also consider checking the actual output of the command to make sure the error is about the namespace not being there anymore.

Reply:
Right, will add checking for service & app presence too before and after.

Reply:
Service and app presence is checked in the `makeApp` and `makeService` functions using `Epinio` func.

Regarding testing after calling deletion of the namespace, I have added a function that checks for the presence of org using `Epinio` func.

Reply:
as long as org "show" action is doing the right thing, it should be ok. An alternative is to check on the kube level (using the `kubectl` helper), that the org namespace is actually gone.

Reply:
Kewl. The `org list` calls kubectl.